sample_consensus/include/pcl/sample_consensus/impl/sac_model_ellipse3d.hpp

```diff
@@ -451,7 +451,7 @@ pcl::SampleConsensusModelEllipse3D<PointT>::optimizeModelCoefficients (
   OptimizationFunctor functor(this, inliers);
   Eigen::NumericalDiff<OptimizationFunctor> num_diff(functor);
   Eigen::LevenbergMarquardt<Eigen::NumericalDiff<OptimizationFunctor>, double> lm(num_diff);
-  Eigen::VectorXd coeff;
+  Eigen::VectorXd coeff = model_coefficients.cast<double>();
   int info = lm.minimize(coeff);
```

Comment:
```suggestion
  auto info = lm.minimize(coeff);
```

Reply:
Why do you propose to change this to auto? Yes, technically it is the enum `LevenbergMarquardtSpace::Status`, but further down in the print, the specifier `%i` expects an int, so casting to int here seems like the right choice.

Reply:
Ahh, yes, then lets just keep it as is.